src/Symfony/Bridge/ProxyManager/README.md

```diff
@@ -9,7 +9,7 @@ Resources
 You can run the unit tests with the following command:
 
     $ cd path/to/Symfony/Bridge/ProxyManager/
-    $ composer.phar install --dev
+    $ composer.phar install
```

Comment:
At least, the .phar is consistent everywhere, but yeah, that makes sense; let's do it.